src/task/CamelJBangTaskProvider.ts

```diff
@@ -59,7 +59,25 @@ export class CamelJBangTaskProvider implements TaskProvider {
 		return undefined;
 	}
 
-	private getCamelJBangVersion(): string {
+	private getCamelJBangCLIVersion(): string {
 		return workspace.getConfiguration().get('camel.debugAdapter.JBangVersion') as string;
 	}
+
+	private getCamelVersion(): string {
+		const camelVersion = workspace.getConfiguration().get('camel.debugAdapter.CamelVersion') as string;
+		if(camelVersion) {
+			return `--camel-version=${camelVersion}`;
+		} else {
+			return '';
+		}
+	}
+
+	private getRedHatMavenRepository(): string {
+		if (this.getCamelVersion().includes('redhat')) {
+			const url = workspace.getConfiguration().get('camel.debugAdapter.RedHatMavenRepository') as string;
+			return `--repos=${url}`;
```

Comment:
we need to use something like `--repos=\$\{repo\},${url}`
it will allow to pick Maven repositories configured globally by the end-user

Reply:
I am having troubles to use it this way 🙁 
![image](https://github.com/camel-tooling/camel-dap-client-vscode/assets/18696866/462f078a-2587-4beb-8856-b79e19dcddcd)

Reply:
```
» camel config list                                                                                                                                                                                                                                                                                                       ~/development/tmp/testing-lsp/testing-commands 130 ↵ 
repos = https://packages.atlassian.com/maven-external
```

Reply:
For me locally the [placeholders](https://camel.apache.org/manual/camel-jbang.html#_placeholders_substitutes) does not work properly or I don't understand how to use them correctly

Reply:
what should happen in case the global value of repos won't be setup? I am not sure if we are able to check the placeholder `$repos` is empty or not before calling in cmd

Reply:
I think this is something that Camel JBang should be able handle

Reply:
@apupier I dont think it is working

Reply:
in the picture you can see I tried to set one repository as global and then run command with another repository which should mean to have both used right?

but in log there is log

```
--repos=,https//....
```

which indicates that `$repos`placeholder is empty probably...

Reply:
raised https://issues.apache.org/jira/browse/CAMEL-19486

Reply:
issue resolved - should be fixed in Camel 3.20.7

the placeholder will be accessible with `#repos` placeholder (the `#` will replace `$`)

Reply:
yeah so we will need to wait for a new release then, and use # in our code

Reply:
ok, in that case I am going to put this PR back to draft and block the related JIRA issue and we will wait for the next release with fix